demisto_sdk/commands/common/update_id_set.py

```diff
@@ -207,12 +207,26 @@ def get_filters_and_transformers_from_playbook(data_dict: dict) -> Tuple[list, l
     all_filters = set()
     all_transformers = set()
 
+    # collect complex values from playbook inputs
+    inputs = data_dict.get('inputs', [])
+    complex_values = [_input.get('value', {}).get('complex', {}) for _input in inputs]
+
     # gets the playbook tasks
     tasks = data_dict.get('tasks', {})
 
+    # collect complex values from playbook tasks
     for task in tasks.values():
         # gets the task value
-        complex_value = task.get('scriptarguments', {}).get('value', {}).get('complex', {})
+        if task.get('type') == 'condition':
+            for condition in task.get('conditions', []):
+                for inner_condition in condition.get('condition', []):
+                    if inner_condition:
+                        complex_values.append(inner_condition[0].get('left', {}).get('value', {}).get('complex', {}))
```

Comment:
are you sure only the first index is relevant?

Reply:
yes, I created a task for testing with one condition that contains 2 conditions inside, as you can see, each condition is a list that contains 1 element:
![image](https://user-images.githubusercontent.com/46249224/125192697-e1981180-e251-11eb-889c-5becfaac7d50.png)
![image](https://user-images.githubusercontent.com/46249224/125192807-4b182000-e252-11eb-93f1-cc067aa30d40.png)

Reply:
try this settings:
![image](https://user-images.githubusercontent.com/30797606/125499932-ea4f9d22-6eb7-45c4-a60f-28d662564610.png)